### ISSUE TITLE
In the cascader component, when set expandTrigger equal to 'hover', loadData function can not trigger?

### ISSUE THREAD

**Issue by ibcLee** [ant-design/ant-design#9084]
### What problem does this feature solve?
I'm not sure if this is a bug

### What does the proposed API look like?
```js
<Cascader
        options={this.state.options}
        loadData={this.loadData}
        onChange={this.onChange}
        expandTrigger="hover"
        changeOnSelect
      />
```
https://codesandbox.io/s/vj85kwq2z5

**Comment by nikogu** [comment]
Can you reproduce on codepen?

**Comment by afc163** [comment]
Fixed in rc-cascader@0.17.2
